README.md

```diff
@@ -172,9 +172,21 @@ it makes this module fairly useless._
 Type: `Object`  
 Default: `null`
 
-If a server instance (eg. Express or Koa) is provided, the `WebSocket` server
+If a server instance (eg. `http.Server`) is provided, the `WebSocket` server
 will attempt to attach to the server instance instead of using a separate port.
 
+This often requires creating the server first and _then_ attaching the middleware
+to the running server, for example:
+
+```js
+// Store a reference to the returned `http.Server`
+const server = koa.listen(port, () => {
+  koa.use(webpackHotClient(compiler, {
+    server, // <-- use the reference above, returned from `koa.listen()`
+  }));
+});
+```
+
```

Comment:
If you'd like to include one-liners below the  for Express, Koa, and `http.Server` that show how to obtain the server instance, that would be acceptable.

Reply:
edit as you see fit - the code example provided is identical for Express or Koa, so I don't personally think it introduces any confusion.

Reply:
I'm sorry but its your responsibility to meet the requests of the review